### PR TITLE
[Sourcemanager] Include a recommendation to user that highlighs the

### PR DESCRIPTION
**Pull request by k0nserv**
importance of updating to the latest version when one is available

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* The notice about a new version being available will now include our
+  recommendation of using the latest version stable version
```
Comment (orta) [inline CHANGELOG.md]: > latest version stable version

**Comment by orta** [comment]
This is good to go after fixing the changelog

**Comment by k0nserv** [comment]
Fixed

cc @orta

**Comment by orta** [comment]
Ace

**Pull request merged**